src/auto/key_file.rs

```diff
@@ -223,7 +223,7 @@ impl KeyFile {
         }
     }
 
-    //pub fn set_boolean_list(&self, group_name: &str, key: &str, list: /*Unimplemented*/&CArray TypeId { ns_id: 0, id: 1 }, length: usize) {
+    //pub fn set_boolean_list(&self, group_name: &str, key: &str, list: /*Unimplemented*/&CArray TypeId { ns_id: 0, id: 1 }) {
```

Comment:
set_string_list() will be fixed with the next GLib .gir update. set_*_list() and get_*_list() requires non-ptr container support.